contrib/generic_proxy/filters/network/source/codecs/dubbo/config.cc

```diff
@@ -35,6 +35,8 @@ absl::string_view responseStatusToStringView(Common::Dubbo::ResponseStatus statu
     ENUM_TO_STRING_VIEW(ClientError);
     ENUM_TO_STRING_VIEW(ServerThreadpoolExhaustedError);
   }
+  static constexpr char notFound[] = "Enum not found";
```

Comment:
Maybe move this out of this function just like the `VERSION_KEY` with value "UnknownResponseStatus"